common/djangoapps/student/migrations/0032_add_display_name_to_userprofile.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2.12 on 2020-04-16 22:36
```

Comment:
**Reminder:** If you've run this migration locally, you'll want to roll it back.  In the future, after a rebase, this will need to become 0033+ because it is in conflict with a new 0032 migration.